src/components/Paginate.tsx

```diff
@@ -33,7 +33,9 @@ export default function Paginate({
       {/* <!-- This example requires Tailwind CSS v2.0+ --> */}
       <div className="px-4 py-3 flex justify-between sm:px-6">
         <div className="flex-1 flex justify-between sm:hidden">
-          {currentPage !== 1 && (
```

Comment:
Before the change in this commit, the "Next" button on small screen sizes would appear on the lefthand side of the screen while on the first page.

![Screen Shot 2020-12-23 at 9 41 05 AM](https://user-images.githubusercontent.com/43425812/103023525-fce43f00-4502-11eb-9f5f-016514c4ae03.png)